pkgs/by-name/ow/owl-lisp/package.nix

```diff
@@ -16,7 +17,17 @@ stdenv.mkDerivation rec {
     hash = "sha256-GfvOkYLo8fgAvGuUa59hDy+sWJSwyntwqMO8TAK/lUo=";
   };
 
-  nativeBuildInputs = [ which ];
+  nativeBuildInputs = [
+    which
+    dash
+  ];
+
+  # Tests fail with bash, replacing with dash seems to work around it
+  # FIXME: Why?
+  prePatch = ''
```

Comment:
We generally do this in `postPatch` instead as it is harder to get 3rd party/upstream patches to apply directly if we have been mucking around with the sources before they are applied.

I know there aren't any `patches` here but it's more about keeping the pattern.